packages/jupytercad-extension/src/mainview.tsx

```diff
@@ -236,9 +236,9 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._scene = new THREE.Scene();
 
-      this._scene.add(new THREE.AmbientLight(0xffffff, 0.8)); // soft white light
+      this._scene.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
 
-      const light = new THREE.SpotLight(0xffffff, 0.2);
+      const light = new THREE.PointLight(0xffffff, 1);
```

Comment:
I did not know adding a point light to the camera would make the point light stick to the camera position! That's good to know!